apps/examples/wifi_manager_sample/wm_test_stress_test1.c

```diff
@@ -23,45 +23,41 @@
 #include <wifi_manager/wifi_manager.h>
 #include <stress_tool/st_perf.h>
 #include "wm_test.h"
+#include "wm_test_log.h"
 
-/*
- * Macro
- */
+/* Macro*/
 #define WM_TEST_TRIAL	   CONFIG_WIFIMANAGER_TEST_TRIAL
```

Comment:
@pillip8282 
CONFIG_WIFIMANAGER_TEST_TRIAL is not defined in all of defconfigs.
You need to add 'CONFIG_WIFIMANAGER_TEST_TRIAL' to all of defconfigs or change this statement to check 'CONFIG_WIFIMANAGER_TEST_TRIAL' exists.